### PR TITLE
Hotfix/channel writer is none

### PR DESCRIPTION
**Pull request by mosquito**
If the channel was closed due to a broken RPC, for example due to Cancelled Error, the Robust Channels instance must be reopened.

### CODE REVIEW

File: aio_pika/channel.py [hunk 1]
```diff
@@ -79,35 +79,51 @@ def return_callbacks(self) -> CallbackCollection:
...
     @property
     def is_closed(self):
-        if not self._channel:
+        if not self._channel or self._is_closed_by_user:
             return True
-        return self.channel.is_closed
+        return self._channel.is_closed
+
+    @property
+    def _is_closed_by_user(self):
```
Comment (decaz) [inline aio_pika/channel.py:87]: Maybe it just worth to add `_is_closed_by_user` attribute for the `Channel` class? I mean changing it manually without all this magic with empty tuple..

**Review by decaz — CHANGES_REQUESTED** [review]

File: aio_pika/channel.py (hunk 1 above)
Comment (mosquito) [reply aio_pika/channel.py:87]: Scheduled on 7.0 in #436

File: tox.ini [hunk 2]
```diff
@@ -1,11 +1,10 @@
 [tox]
-envlist = lint,py3{5,6,7},py3{5,6,7}-{uvloop}
+envlist = lint,py3{6,7},py3{6,7}-{uvloop}
```
[the remark this answers is not in the document]
Comment (mosquito) [reply tox.ini:2]: Unfortunately, if you look at the ratings, 3.6 is the most popular Python on this planet.

File: tox.ini [hunk 3]
```diff
@@ -15,15 +14,15 @@ extras =
   develop
 
 commands=
-  py.test -vv --cov=aio_pika --cov-report=term-missing --doctest-modules --aiomisc-test-timeout=30 tests
+  py.test -sxv --cov=aio_pika --cov-report=term-missing --doctest-modules --aiomisc-test-timeout=30 tests
```
Comment (mosquito) [inline tox.ini:17]: Ooops. Should be reverted.

File: tox.ini (hunk 2 above)
[the remark this answers is not in the document]
Comment (decaz) [reply tox.ini:2]: Ok, let's just add latest versions.

[the remark this answers is not in the document]
Comment (decaz) [reply tox.ini:2]: @mosquito I see there is tox.ini at the root and tox.ini at .github/workflows, - should we need the first file if GitHub Actions are already in use.

**Pull request merged**